go.mod

```diff
@@ -279,4 +279,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
-replace github.com/go-git/go-git/v5 => github.com/argoproj-labs/go-git/v5 v5.4.7
+replace (
+	github.com/antonmedv/expr => github.com/expr-lang/expr v0.0.0-20230912141041-709c5dd55aa7
```

Comment:
Confirmed that this commit SHA matches the one I confirmed to be from the original repo in https://github.com/argoproj/argo-workflows/discussions/11964#discussioncomment-7232791. 

Per my comment there though, we do not yet have confirmation from the author that [`expr-lang/expr`](https://github.com/expr-lang/expr) is an "official" fork. I think we should wait for confirmation from the author or use a known, trusted fork in the interim (such as [my fork](https://github.com/agilgur5/expr))